Meadowlark-js/packages/meadowlark-core/src/handler/FrontendFacade.ts

```diff
@@ -34,6 +38,16 @@ const moduleName = 'core.handler.FrontendFacade';
  */
 async function initialize(): Promise<void> {
   initializeLogging();
+  const sdk = new NodeSDK({
+    traceExporter: new ConsoleSpanExporter(),
+    metricReader: new PeriodicExportingMetricReader({
+      exporter: new ConsoleMetricExporter(),
+    }),
+    instrumentations: [getNodeAutoInstrumentations()],
+  });
+
+  sdk.start();
```

Comment:
Since this is a design spike, perhaps this code should be removed.

Reply:
Yep, that was accidental, removing